src/Ripple/TouchRipple.js

```diff
@@ -26,6 +26,9 @@ export const styleSheet = createStyleSheet('MuiTouchRipple', () => ({
 
 export default class TouchRipple extends Component {
   static propTypes = {
+    /**
+     * TODO: What is this?
```

Comment:
It's determining if the ripple start at the center of the component or at the mouse interaction location.